mesonbuild/interpreter.py

```diff
@@ -3095,6 +3112,9 @@ def find_program_impl(self, args, for_machine: MachineChoice = MachineChoice.HOS
                 return ExternalProgramHolder(dependencies.NonExistingExternalProgram())
         return progobj
 
+
+
+    @FeatureNewKwargs('find_program', '???', ['fallback'])
```

Comment:
__[Flake8]__ too many blank lines (3) ([see on Sider](https://sider.review/gh/repos/19784232/pulls/6472#issue-5064127))

<details>

| Rule |
| - |
| E303 |

</details>
<sub>posted by Sider</sub>

Reply:
0.54.0 for now.